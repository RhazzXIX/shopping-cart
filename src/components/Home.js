import "../styles/Home.css";

const Home = () => {
  return (
    <main id="home">
      <h1>Hoarder Digital</h1>
      <p>Photo by: <a href="https://www.pexels.com/@n-voitkevich/">Nataliya Vaitkevich</a></p>
    </main>
  );
};

export default Home;
