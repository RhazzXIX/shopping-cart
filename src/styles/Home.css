main#home {
  flex-grow: 1;
  background: url("../images/shoppingBG.webp") no-repeat center;
  background-size: cover;
  min-height: 818px;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  align-items: center;
}

main#home h1 {
  margin: 30px auto;
  padding: 4px 8px;
  border-radius: 12px;
  text-shadow: 0 0 4px black;
  letter-spacing: 2px;
  width: max-content;
}

main#home 
p {
  background-color: rgba(255, 255, 255, 0.6);
  width: max-content;
  padding: 2px 8px;
  border-radius: 12px;
  margin-bottom: 12px;
}

main#home 
p:hover {
  background-color: white;
  box-shadow: 0 0 4px black;
}
