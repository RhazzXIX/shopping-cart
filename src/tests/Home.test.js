import { render } from "@testing-library/react";
import Home from "../components/Home";

test("renders the home component", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
