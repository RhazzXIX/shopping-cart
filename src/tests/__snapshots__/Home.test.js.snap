// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the home component 1`] = `
<div>
  <main
    id="home"
  >
    <h1>
      Hoarder Digital
    </h1>
    <p>
      Photo by: 
      <a
        href="https://www.pexels.com/@n-voitkevich/"
      >
        Nataliya Vaitkevich
      </a>
    </p>
  </main>
</div>
`;
